zebra-chain/src/primitives/redpallas/constants.rs

```diff
@@ -11,12 +11,12 @@
 ///
 /// [pallasandvesta]: https://zips.z.cash/protocol/nu5.pdf#pallasandvesta
 pub const SPENDAUTHSIG_BASEPOINT_BYTES: [u8; 32] = [
-    215, 148, 162, 4, 167, 65, 231, 17, 216, 7, 4, 206, 68, 161, 32, 20, 67, 192, 174, 143, 131,
-    35, 240, 117, 113, 113, 7, 198, 56, 190, 133, 53,
+    99, 201, 117, 184, 132, 114, 26, 141, 12, 161, 112, 123, 227, 12, 127, 12, 95, 68, 95, 62, 124,
+    24, 141, 59, 6, 214, 241, 40, 179, 35, 85, 183,
 ];
 
 /// The byte-encoding of the basepoint for `BindingSig` on the Pallas curve.
 pub const BINDINGSIG_BASEPOINT_BYTES: [u8; 32] = [
-    48, 181, 242, 170, 173, 50, 86, 48, 188, 221, 219, 206, 77, 103, 101, 109, 5, 253, 28, 194,
-    208, 55, 187, 83, 117, 182, 233, 109, 158, 1, 161, 215,
+    145, 90, 60, 136, 104, 198, 195, 14, 47, 128, 144, 238, 69, 215, 110, 64, 72, 32, 141, 234, 91,
+    35, 102, 79, 187, 9, 164, 15, 85, 68, 244, 7,
 ];
```

Comment:
These seem to duplicate the `lazy_static`s in:
https://github.com/ZcashFoundation/zebra/blob/566a10151ca8ab40b4568ab75c2bfaa5881618b6/zebra-chain/src/orchard/commitment.rs#L324

Reply:
This is interesting; so what I may do is swap what we have, but keep the test cases that assert that the basepoints calculated for spendauth and binding and `lazy_static!`'d match the bytes we expect.

Reply:
> These seem to duplicate the `lazy_static`s in:
> https://github.com/ZcashFoundation/zebra/blob/566a10151ca8ab40b4568ab75c2bfaa5881618b6/zebra-chain/src/orchard/commitment.rs#L324

Oh yes the Binding Sig basepoint is a duplicate, correct.  The SpendAuth basepoint is distinct from V in the value commitments

Reply:
So originally, had in mind that any `redpallas` would become a separate crate/repo like `redjubjub` is, so deduping some of these constants may not help there, it wouldn't be hard to split back out again, but

Reply:
Whatever you think is best here, as long as there are tests to make sure they all match

Reply:
If there isn't much movement to extract redpallas in the near future, I might reorganize several of these shared constants or make them more pub(crate) so we can check cross-compatibility.